README.md

```diff
@@ -1,58 +1,57 @@
 expRESTive [![Build Status](https://travis-ci.org/alexdavid/exprestive.svg)](https://travis-ci.org/alexdavid/exprestive) [![Dependency Status](https://david-dm.org/alexdavid/exprestive.png)](https://david-dm.org/alexdavid/exprestive)
 ==========
 
-Expressive RESTful express router
+A RESTful routing middleware for [Express.js](http://expressjs.com).
+
 
 ## Basic usage
-* Install [Express](http://expressjs.com/) and Exprestive
+Assuming you have already created an [Express](http://expressjs.com/) application
+by following the [Express installation instructions](http://expressjs.com/starter/installing.html).
+Now:
 
-```shell
-$ npm install --save express exprestive
-```
+* add expRESTive to your _package.json_ file: `$ npm install --save express exprestive`
```

Comment:
Can probably remove the `express` install here since we assume an existing express application above.